index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="./styles/defaultStyle.css">
+	<link rel="stylesheet" href="./styles/generalStyle.css">
+	<link rel="stylesheet" href="./styles/style.css">
+	<title>DiegoPortfolio</title>
+</head>
+
+<body>
+	<div class="firstSection">
+		<nav>
+			<ul class="toolbar">
+				<li>John Doe</li>
+				<li><img src="./icons/iconMenu.png" alt="iconMenu"></li>
+			</ul>
+		</nav>
+		<div class="headline">
+			<h1>Hey There. I’m Diego</h1>
+			<h2>I’am a Software Developer</h2>
+			<p>I can help you build a product , feature or website Look through some of my work and experience! If you like
+				what you see and have a project you need coded,
+				don’t hestiate to contact me.
+			</p>
+		</div>
```

Comment:
- [ ] Great job using the `<div>` tag 💯 , but I think it could be improved if you use a `<section<` tag. 🦓  It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info. Kindly change `<div>` tag to `<section>` tag. 🍐